contracts/context-config/src/mutate.rs

```diff
@@ -15,6 +16,7 @@ use super::{
 
 #[near]
 impl ContextConfigs {
+    #[payable]
```

Comment:
deployment doesn't work otherwise? Can we somehow make this charge the context config account instead of the transaction author?